src/shared/schemas.ts

```diff
@@ -44,6 +44,10 @@ export interface SchemaHandler {
     isMapped(f: vscode.Uri | string): boolean
 }
 
+export const cfnSchemaUri = (path: vscode.Uri) => vscode.Uri.joinPath(path, 'cloudformation.schema.json')
+export const samSchemaUri = (path: vscode.Uri) => vscode.Uri.joinPath(path, 'sam.schema.json')
+export const buildSpecSchemaUri = (path: vscode.Uri) => vscode.Uri.joinPath(path, 'buildspec.schema.json')
```

Comment:
How likely do you think additional schemas will be added in the near-future? If the chances are pretty high then further generalizations might be justified.

Reply:
I don't think we'll have any additional schemas here anymore. They should all come through the language server and we can eventually get rid of most of the schema related code